src/Symfony/Bridge/Monolog/Handler/ElasticsearchLogstashHandler.php

```diff
@@ -0,0 +1,169 @@
+    }
+
+    public function handleBatch(array $records): void
+    {
+        $records = array_filter($records, function (array $record) {
+            return $this->isHandling($record);
+        });
+
+        if (!$records) {
+            return;
+        }
+
+        $this->sendToElasticsearch($records);
+    }
+
+    protected function getDefaultFormatter(): FormatterInterface
+    {
+        return new LogstashFormatter('application', null, null, 'ctxt_', LogstashFormatter::V1);
+    }
+
+    private function sendToElasticsearch(array $records)
+    {
+        $formatter = $this->getFormatter();
+
+        $body = '';
+        foreach ($records as $record) {
+            if ($this->processors) {
+                foreach ($this->processors as $processor) {
+                    $record = \call_user_func($processor, $record);
+                }
+            }
+
+            $body .= json_encode([
+                'index' => [
+                    '_index' => $this->index,
+                    '_type' => '_doc',
+                ],
+            ]);
+            $body .= "\n";
+            $body .= $formatter->format($record);
+            $body .= "\n";
+        }
+
+        $response = $this->client->request('POST', $this->endpoint.'/_bulk', [
+            'body' => $body,
+            'headers' => [
+                'Content-Type' => 'application/json',
+            ],
+        ]);
+
+        $this->responses->attach($response);
+
+        try {
+            $this->wait(0.0, false);
+        } catch (ExceptionInterface $e) {
+            error_log(sprintf("Could not push logs to elasticsearch:\n%s", (string) $e));
+        }
+    }
+
+    public function __destruct()
+    {
+        try {
+            $this->wait(null, true);
+        } catch (ExceptionInterface $e) {
+            error_log(sprintf("Could not push logs to elasticsearch:\n%s", (string) $e));
+        }
+    }
+
+    private function wait(?float $timeout, bool $errorOnTimeout)
+    {
+        $e = null;
+        $remainingTimeout = $timeout;
+
+        if (!$errorOnTimeout && $remainingTimeout) {
+            $startTime = microtime(true);
+        }
+
+        foreach ($this->client->stream($this->responses, $remainingTimeout) as $response => $chunk) {
+            try {
+                if ($chunk->isTimeout() && !$errorOnTimeout) {
+                    continue;
+                }
+                if (!$chunk->isFirst() && !$chunk->isLast()) {
+                    continue;
+                }
+                if ($chunk->isLast()) {
+                    $this->responses->detach($response);
+                }
+            } catch (ExceptionInterface $e) {
+                $this->responses->detach($response);
+            } finally {
+                if (!$errorOnTimeout && $remainingTimeout) {
+                    $remainingTimeout = max(0.0, $timeout - microtime(true) + $startTime);
+                }
+            }
+        }
+
+        if ($e) {
+            throw $e;
```

Comment:
can be replaced by `error_log(sprintf("Could not push logs to elasticsearch:\n%s", (string) $e));` and the try/catch be removed above